lib/web_ui/lib/src/engine/window.dart

```diff
@@ -202,15 +202,32 @@ class EngineFlutterWindow extends ui.SingletonFlutterWindow {
     });
   }
 
-  @override
-  ui.ViewConfiguration get viewConfiguration {
+  ViewConfiguration get _viewConfiguration {
     final EnginePlatformDispatcher engineDispatcher =
         platformDispatcher as EnginePlatformDispatcher;
     assert(engineDispatcher.windowConfigurations.containsKey(viewId));
     return engineDispatcher.windowConfigurations[viewId] ??
-        const ui.ViewConfiguration();
+        const ViewConfiguration();
   }
 
+  @override
+  ui.Rect get physicalGeometry => _viewConfiguration.geometry;
```

Comment:
I suspect some of these should come from the DimensionsProvider. Where can I read more about this new batch of view attributes?

Reply:
They used to be implemented by the base FlutterView class in `lib/web_ui/lib/window.dart`. Since that one is now fully abstract, I moved the exact same implementation into the concrete class here. There should be no difference in where the value is coming from compared to what it was before. I am actually not familiar with the `DimensionsProvider`.

Reply:
Oh, and these are documented on FlutterView, but they have been around for some time: https://master-api.flutter.dev/flutter/dart-ui/FlutterView-class.html

Reply:
Totally missed that these came from the default implementation, I guess web is not relying in any of these (for now :P)